### DISCUSSION TITLE
会場レイアウトを決める

### DISCUSSION THREAD

**Discussion by yebis0942** [kyoto-go/go-conference-mini-2023#11] [General]
[会場レイアウト図](https://docs.google.com/drawings/d/11ymFstX4t15zxgtiHZe7pAIRFCc_FFBgPXF95fOSJjw/edit)

**Comment by yebis0942** [comment]
ゴミ袋の種類は？

**Reply by yebis0942** [reply]
会場の分別ルールは

* ペットボトル
* 缶、ビン
* 燃えるゴミ

とのこと。

[京都市の分別ルール](https://www.city.kyoto.lg.jp/menu1/category/1-3-9-0-0-0-0-0-0-0.html)と違うのでujiさんに会場の方に確認していただいています。

**Reply by chiroruxx** [reply]
ダンボールをゴミとして出せるか気になりますね。

**Reply by uji** [reply]
回収いただける分類は以下で、京都市のものとは別とのことでした
- ペットボトル
- 缶、ビン
- 燃えるゴミ

プラスチックや段ボールは回収する場合自分達で処理する必要がありそうですね

**Reply by chiroruxx** [reply]
> プラスチックや段ボールは回収する場合自分達で処理する必要がありそうですね

なるほどですねー。スポンサーさんいないとのことなので参加者からダンボールが出ることは無さそうですが、当日に消費するものがあるとダンボール発生するので気をつける必要がありそうですね。（参加者に配布するものなど）

**Comment by yebis0942** [comment]
荷物置き場の利用対象者と配置は？

**Reply by yebis0942** [reply]
:memo: トラブルが増えそうなので利用者は登壇者やスタッフのみに限定するのが良さそうとの意見が多い。

**Reply by yebis0942** [reply]
登壇者とスタッフのみとするように更新しました。

**Comment by yebis0942** [comment]
控室の用途は？

**Reply by yebis0942** [reply]
* 登壇前に登壇者とやりとりをするスペースとして使う
    * 登壇者がいるかどうかの確認
    * Stream Yardへの招待など
* 準備が終わったら会場で待機してもらう

**Comment by yebis0942** [comment]
小会議室の用途は？

**Reply by yebis0942** [reply]
運営スタッフ用のエリアとして物品等の保管に利用する。

**Comment by yebis0942** [comment]
ホワイトボードの位置と用途は？

**Reply by chiroruxx** [reply]
他のカンファレンスでは以下のような用途で使われてました。

- アンカンファレンスの募集
- 懇親会LTの募集
- フリージョブボード(誰でも求人を書いていい)
- フリーステッカーボード(誰でもステッカーを貼れる。ボードに直接ではなく、大きめの紙をボードにかけておき、それに貼る)

**Reply by yebis0942** [reply]
今回はアンカンファレンスや懇親会LTは実施予定なさそうなので、ジョブボードやステッカーボードなどの用途を想定して配置してみます :pray:

**Comment by uji** [comment]
イベントの内容的にそこまでゴミは発生しないはずなので、ゴミ箱は多いかもしれません
- メイン会場1個
- 受付付近1個(帰りに名札を捨てる用)

くらいでも良いのかなと思いました

**Reply by yebis0942** [reply]
たしかに。メイン会場のゴミ箱は見渡したときに視界に入りやすく、出入り口から遠い左下に配置してみました。

**Comment by uji** [comment]
お土産コーナーの場所
メイン会場左下あたりが良さそう？

**Reply by yebis0942** [reply]
会場左側がよさそうですよね。

ためしに入口からパッと見えやすそうなコモンズ01左上の角に配置してみましたが、どうでしょうか。

**Comment by yebis0942** [comment]
参加者向けのコンセントは提供する？

**Reply by yebis0942** [reply]
> 延長コードは無料で貸出可能ですが、在庫に限りがあるため「1団体様につき2本まで」としています。それ以上必要となる場合はご自身でお持ち込みください。 

https://question.kyoto-shinkin.co.jp/about/faq/

とのことなので、会場で貸し出していただける延長コードは配信ブースだけで使い切りそう。

**Reply by yebis0942** [reply]
十分な数の電源タップを用意するのが難しいため、参加者向けには電源を提供しないことにしました。

ref.
- https://scrapbox.io/kyotogo/2023%2F11%2F19_Go_Conference_mini%E6%89%93%E3%81%A1%E5%90%88%E3%82%8F%E3%81%9B
- https://scrapbox.io/kyotogo/2023%2F11%2F12_Go_Conference_mini%E6%89%93%E3%81%A1%E5%90%88%E3%82%8F%E3%81%9B
